include/exadg/incompressible_navier_stokes/postprocessor/output_generator.cpp

```diff
@@ -452,11 +452,9 @@ OutputGenerator<dim, Number>::calculate_additional_fields(VectorType const & vel
 
     if(output_data.write_cfl)
     {
-      auto param = navier_stokes_operator->get_param();
-      auto time_step_size =
-        navier_stokes_operator->calculate_time_step_cfl(velocity,
-                                                        param.cfl,
-                                                        param.cfl_exponent_fe_degree_velocity);
+      auto   param          = navier_stokes_operator->get_param();
```

Comment:
Should this be a reference?

Reply:
I think this is ok. 

I see another problem here. The result is not correct if one specifies temporal refinements in the input file. So we should find another solution for this.
@bergbauer  A relative CFL number should be enough/sufficient for this kind of result? So we could calculate the time step size with `cfl=1`, and afterwards compute the dof-vector of cfl-values, which takes `cfl=1` in one cell and smaller values in other cells. I guess we could then remove the getter `navier_stokes_operator->get_garam()`.

Reply:
... should be done in a separate PR.

Reply:
The intention of this output is to see where in the geometry/mesh the CFL condition restricts our time step. So yes, a relative measure is sufficient but should be named appropriately.

Reply:
Regarding my understanding of
> Should this be a reference?

I think that the copy is expensive (in relative terms, it is "only" several dozen of instructions), compared to the single variable `cfl` that you want to retrieve. I would personally prefer to simply write `time_step_size *= navier_stokes_operator->get_param();` in the use in line 457 below. But I guess this will be refactored anyway, so let's keep it for now.

Reply:
yes, see #108